json/public.go

```diff
@@ -5,7 +5,7 @@ package json
 
 import (
 	"fmt"
-	"io/ioutil"
+	"io"
 	"os"
```

Comment:
Fixes:
```
json/public.go:8:2: SA1019: "io/ioutil" has been deprecated since Go 1.19: As of Go 1.16, the same functionality is now provided by package [io] or package [
os], and those implementations should be preferred in new code. See the specific function documentation for details. (staticcheck)
        "io/ioutil"
```